### DISCUSSION TITLE
Welcome to ChestsPlusPlus Discussions!

### DISCUSSION THREAD

**Discussion by JamesPeters98** [ChestsPlusPlus/ChestsPlusPlus#26] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DexterXS** [comment]
Hello! You can add my translate file?
[uk_UA.txt](https://github.com/ChestsPlusPlus/ChestsPlusPlus/files/11714746/uk_UA.txt)

**Comment by jiuwu02** [comment]
I made Chinese translation file here, please help me to add to the plugin
[zh_CN.txt](https://github.com/ChestsPlusPlus/ChestsPlusPlus/files/12850692/zh_CN.txt)

**Reply by DexterXS** [reply]
still need help?

**Reply by DexterXS** [reply]
There you need a plugin file that you will cancel (using the archiver) and put your translation there
